src/main.rs

```diff
@@ -452,6 +470,19 @@ async fn main() -> Result<(), ServerError> {
         }
     });
 
+    #[cfg(feature = "search")]
+    {
+        let search_arguments = SearchArguments {
+            api_key: cli.api_key,
+            query_server_url: cli.query_server_url,
+            search_backend: cli.search_backend,
+        };
+
+        SEARCH_ARGUMENTS
+            .set(search_arguments)
+            .map_err(|_| ServerError::Operation("Failed to set `SERVER_INFO`.".to_string()))?;
```

Comment:
`SERVER_INFO` in the error message should be `SEARCH_ARGUMENTS`, correct?

Reply:
Yes, fixing this.